src/iomei.cpp

```diff
@@ -4508,13 +4510,12 @@ bool MeiInput::ReadSyl(Object *parent, pugi::xml_node syl)
 {
     Syl *vrvSyl = new Syl();
     ReadLayerElement(syl, vrvSyl);
+    ReadFacsimileInterface(syl, vrvSyl);
```

Comment:
Note: It won't be necessary to specify this as in the PR the `LayerElement` class has a `FacsimileInterface`. So `ReadLayerElement` would also call `ReadFacsimileInterface`.